modules/chapter4/pages/section5.adoc

```diff
@@ -0,0 +1,143 @@
+= Install Sample Application on Tenant Cluster
+:experimental:
+
+In this section, you will be installing basic _Node.js_ application on _Tenant_ cluster.
+You will also be testing the high availability and resilience of deployed _Node.js_ application.
+
+image::MCAP_setup_1.png[]
+
+== Prerequisites
+
+. Ensure _OpenShift Data Foundation_ operator is installed.
+
+. Ensure _Image Registry_ operator is using `Noobaa` storage with Red Hat OpenShift Data Foundation.
++
+.Sample output:
+----
+[root@hypervisor ~]# oc get co image-registry
+NAME             VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
+image-registry   4.16.8    True        False         False      61s
+----
+
+== Install _Node.js_ application on _Tenant_ cluster
+
+. From the left navigation pane, click menu:Administrator[Developer].
++
+image::tenant_console_switch_view.png[]
+
+. You will be redirected to developer view.
++
+Click on text btn:[create a Project].
++
+image::tenant_console_developer_view.png[]
+
+. In the create project window, provide the project details.
++
+Name: `sample`.
++
+Display name: `sample`.
++
+Description: `This is sample project`.
++
+Click btn:[Create] to create the project.
++
+image::tenant_console_create_project.png[]
+
+. In the project view, to add the sample application click on text btn:[Add page].
++
+image::tenant_console_nodejs_app_add.png[]
+
+. In the add page window, click on text btn:[View all samples].
++
+image::tenant_console_nodejs_app_sample.png[]
+
+. In the search section, search for `basic node`.
++
+Select the `Basic Node.js` application from samples.
++
+image::tenant_console_nodejs_app_search.png[]
+
+. In `Basic Node.js` application building window, keep all options as is and click btn:[Create] to create the application.
++
+image::tenant_console_nodejs_app_create.png[]
+
+. In topology view, you will see application is created.
++
+Click on the circle to view application overview.
```

Comment:
"Click on the circle to view the application overview."

Reply:
Done.